docs/content/mongocxx-v3/installation.md

```diff
@@ -118,92 +118,126 @@ obtain.
 
 ### Step 4: Configure the driver
 
-On Unix systems, `libmongoc` installs into `/usr/local` by default.  To
-configure `mongocxx` for installation into `/usr/local` as well, use the
-following `cmake` command:
-
-(***NOTE***: The trailing `..` below is important!  Don't omit it.)
+On Unix systems, `libmongoc` installs into `/usr/local` by default. Without additional
+configuration, `mongocxx` installs into its local build directory as a courtesy to those who build
+from source. To configure `mongocxx` for installation into `/usr/local` as well, use the following
+`cmake` command:
 
 ```
-cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local ..
+cmake ..                                \
+    -DCMAKE_BUILD_TYPE=Release          \
+    -DCMAKE_INSTALL_PREFIX=/usr/local
+```
+
+In the Unix examples that follow,
+`mongocxx` is customized in these ways:
+* `libmongoc` is found in `/opt/mongo-c-driver`.
+* `mongocxx` is to be installed into `/opt/mongo-cxx-driver`.
+
+With those two distinct (arbitrary) install locations, a user would run this `cmake` command:
+```sh
+cmake ..                                            \
+    -DCMAKE_BUILD_TYPE=Release                      \
+    -DCMAKE_PREFIX_PATH=/opt/mongo-c-driver         \
+    -DCMAKE_INSTALL_PREFIX=/opt/mongo-cxx-driver
 ```
 
-`/usr/local` may be replaced with the directory that `libmongoc` was
-installed to.
+> *Note* If you need multiple paths in a CMake PATH variable, separate them with a semicolon like
+> this:
+> `-DCMAKE_PREFIX_PATH="/your/cdriver/prefix;/some/other/path"`
```

Comment:
Oh, that's a nice reminder.

Reply:
I am a bit late on this comment, but I believe that semi-colon is Windows-specific.  For Unix-like systems a colon would be used instead.

Reply:
Actually this seems to work for me (on macOS at least). https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html says it's a "Semicolon-separated list".

Reply:
Interesting...I seemed to recall that it used the platform-specific separator and so use colon on Linux.  Of course, given the very cross-platform nature of CMake it is not all that surprising to find that it can intelligently use either and convert as a appropriate.  Feel free to disregard my earlier comment.

Reply:
So it's actually the list separator in CMake. IIRC, it works like this:
```
set(A B C D)
message(FATAL_ERROR "${A}")
// Prints B;C;D
```